packages/components/src/Icons/index.ts

```diff
@@ -43,5 +42,4 @@ export const MYSQL_ICON = MySqlIcon;
 export const NGINX_ICON = NginxIcon;
 export const SYSLOG_ICON = SyslogIcon;
 export const CALYPTIA_LOGO_NAME = CalyptiaLogoName;
-export const VIVO_LOGO = VivoIcon;
```

Comment:
We are only using two icons, so I put them in their respective components. However I didn't clean up yet since I figured I could do it as part of the asset reduction ticket. 

I cleaned it up here: 7b41ed61ef0193a6cf3a219f46f12ce0f0eda74a